js/react_native/android/src/main/java/ai/onnxruntime/reactnative/OnnxruntimeModule.java

```diff
@@ -270,6 +300,9 @@ public WritableMap run(String key, ReadableMap input, ReadableArray output, Read
 
     } finally {
       OnnxValue.close(feed);
+      if (result != null) {
+        result.close();
+      }
```

Comment:
The output values isn't released after session closed. Not sure if I'm doing this right.